Packs/Elasticsearch/Integrations/Elasticsearch_v2/README.md

```diff
@@ -49,36 +53,36 @@ Queries an index.
 
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| index | The index in which to perform a search. | Required |
-| query | The string to query (in Lucene syntax). Possible values are: . | Optional |
-| fields | A comma-separated list of document fields to fetch. If empty, the entire document is fetched. | Optional |
-| explain | Calculates an explanation of a score for a query. For example, "value:1.6943597". Possible values are: true, false. Default is false. | Optional |
-| page | The page number from which to start a search. Default is 0. | Optional |
-| size | The number of documents displayed per page. Can be an integer between "1" and "10,000". Default is 100. | Optional |
-| sort-field | The field by which to sort the results table. The supported result types are boolean, numeric, date, and keyword fields. Keyword fields require the doc_values parameter to be set to "true" from the Elasticsearch server. Possible values are: . | Optional |
-| sort-order | The order by which to sort the results table. The results tables can only be sorted if a sort-field is defined. Possible values are: asc, desc. Default is asc. | Optional |
-| query_dsl | Will overwrite the ‘query' arguments. | Optional |
-| timestamp_range_start | The starting time of the time range. | Optional |
-| timestamp_range_end | The ending time of the time range. | Optional |
-| timestamp_field | Timestamp field name. Default is @timestamp. | Optional |
+| index | The index in which to perform a search. | Required | 
+| query | The string to query (in Lucene syntax). Possible values are: . | Optional | 
+| fields | A comma-separated list of document fields to fetch. If empty, the entire document is fetched. | Optional | 
+| explain | Calculates an explanation of a score for a query. For example, "value:1.6943597". Possible values are: true, false. Default is false. | Optional | 
```

Comment:
Generates an explanation of a score for a query, for example, "value:1.6943597". 

(Remove **Possible values are: true, false. Default is false.**)